bot_modules/faceit.js

```diff
@@ -126,15 +122,12 @@ module.exports = {
       callback(true);
     }
   },
-  RemoveBan(userLink, user) {
+  RemoveBan(userLink) {
     // DELETE https://api.faceit.com/hubs/v1/hub/{hubId}/ban/{userId}
     // Authorization: Bearer {userToken}
     let userId = GetUserToken(userLink);
```

Comment:
```suggestion
    let userId = await this.GetUserToken(userLink);
```
La fonction GetUserToken est mal appelée.